### DISCUSSION TITLE
dataframe with build times?

### DISCUSSION THREAD

**Discussion by joelnitta** [ropensci/targets#486] [Help] [answered]
Sorry if this is already documented somewhere, but I couldn't find anything. `drake` had [`build_times()`](https://docs.ropensci.org/drake/reference/build_times.html). Is there a similar function (or other way to do the equivalent thing) in `targets`?

**Comment by gorkang** [comment, marked as answer]
Is this what you need?

```
  targets::tar_meta(fields = seconds) # Dataframe with seconds it took each target
  targets::tar_visnetwork(label = "time") # Visual representation including time below each target
```

**Reply by joelnitta** [reply]
Thanks! `targets::tar_meta(fields = seconds)` is indeed what I was looking for. I'm running this on a server with no easy way to view the output of `targets::tar_visnetwork()`, so that's why I just wanted the dataframe.
